test/libsolidity/syntaxTests/events/event_selector_library.sol

```diff
@@ -0,0 +1,17 @@
+library Y {
+    event E() anonymous;
+}
+
+contract C {
+    function test1() external pure returns (bytes32) {
+        return Y.E.selector;
+    }
+
+    bytes32 s5 = Y.E.selector;
+
+    function test2() view external returns (bytes32) {
+        return s5;
+    }
+}
+// ----
+// TypeError 9582: (123-135): Member "selector" not found or not visible after argument-dependent lookup in function ().
```

Comment:
Is it on purpose that `.selector` is used twice but there is only a single error? Which one are you testing? Is one of them working?

Reply:
Both the selector calls should not be working. I've confirmed this after removing each function one by one and checking for syntax errors in the other.

But it's printing only 1, presumably because the code stops after finding the first syntax error.

I'll split the tests into two files and push changes